internal/jobcontainers/jobcontainer.go

```diff
@@ -314,11 +318,23 @@ func (c *JobContainer) Start(ctx context.Context) error {
 	return nil
 }
 
-// Close closes any open handles.
+// Close free's up any resources (handles, temporary accounts).
 func (c *JobContainer) Close() error {
 	if err := c.job.Close(); err != nil {
 		return err
 	}
+
+	if err := c.token.Close(); err != nil {
+		return fmt.Errorf("failed to close token: %w", err)
+	}
+
+	// Delete the containers local account if one was created
+	if c.localUserAccount != "" {
+		if err := winapi.NetUserDel("", c.localUserAccount); err != nil {
+			return fmt.Errorf("failed to delete local user account: %w", err)
+		}
+	}
+
```

Comment:
Should this be best effort?

Reply:
As in don't return the error and just log it?

Reply:
Yeah

Reply:
Mm we could. I suppose it's not exactly catastrophic.